### PR TITLE
Psmdb 1376 release notes 7.0.4

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

**Review by igorsol — APPROVED** [review]
LGTM

File: mkdocs-base.yml [hunk 1]
```diff
@@ -117,7 +118,7 @@ plugins:
   with-pdf: # https://github.com/orzih/mkdocs-with-pdf
           output_path: '_pdf/PerconaServerforMongoDB-7.0.pdf'
           cover_title: 'Server for MongoDB Documentation 7.0'
-          cover_subtitle: 7.0.2-1 (October 5, 2023)
+          cover_subtitle: 7.0.3-2 (December 11, 2023)
```
Comment (sandraromanchenko) [inline mkdocs-base.yml]:
```suggestion
          cover_subtitle: 7.0.4-2 (December 11, 2023)
```

File: docs/release_notes/index.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 
 # Percona Server for MongoDB 7.0 release notes
 
+* [Percona Server for MongoDB 7.0.4-2 (2023-12-11)](7.0.3-2.md)
```
Comment (sandraromanchenko) [inline docs/release_notes/index.md]:
```suggestion
* [Percona Server for MongoDB 7.0.4-2 (2023-12-11)](7.0.4-2.md)
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

**Comment by kwilktx** [comment]
@nastena1606 @igorsol @sandraromanchenko 

I think we need to mention the Downgrade and fCV changes and the Backward-Incompatible Features - It is too big a thing not to mention - with consequences

Starting in MongoDB 7.0:
    Binary downgrades are no longer supported for MongoDB Community Edition.
    
    The [setFeatureCompatibilityVersion](https://www.mongodb.com/docs/v7.0/reference/command/setFeatureCompatibilityVersion/#mongodb-dbcommand-dbcmd.setFeatureCompatibilityVersion) command requires an additional parameter, confirm, which must be set to true to upgrade or downgrade fCV.
    
    https://www.mongodb.com/docs/v7.0/release-notes/7.0/#downgrade-considerations
    
https://www.mongodb.com/docs/v7.0/release-notes/7.0-compatibility/#std-label-7.0-downgrade-incompatible

wildcard indexes, config servers that store data, columnar indexes that would have to be removed.

Other than that the bugs that you mention as fixed look good.

File: docs/release_notes/7.0.4-2.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    - After you have upgraded the database, allow your deployment to run without setting [setFeatureCompatibilityVersion](https://www.mongodb.com/docs/manual/reference/command/setFeatureCompatibilityVersion/#mongodb-dbcommand-dbcmd.setFeatureCompatibilityVersion) to 7.0 for a burn-in period to ensure the likelihood of downgrade is minimal. When you are confident that the likelihood of downgrade is minimal, enable these features.
+    - The [setFeatureCompatibilityVersion](https://www.mongodb.com/docs/manual/reference/command/setFeatureCompatibilityVersion/#mongodb-dbcommand-dbcmd.setFeatureCompatibilityVersion) command requires an additional parameter, `confirm`. You must set it to `true` to upgrade or downgrade fCV.
+
+* Due to [time series sharded cluster operation routing issue](https://jira.mongodb.org/browse/SERVER-80203) occurred in MongoDB version 7.0.1 - 7.0.3 and fixed in version 7.0.4, follow closely the upstream remediation recommendations to identify and recover orphanated collections.
```
Comment (sandraromanchenko) [inline docs/release_notes/7.0.4-2.md:21]:
```suggestion
* Due to [time series sharded cluster operation routing issue](https://jira.mongodb.org/browse/SERVER-80203) occurred in MongoDB version 7.0.1 - 7.0.2 and fixed in version 7.0.3, follow closely the upstream remediation recommendations to identify and recover orphanated collections.
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

**Review by sandraromanchenko — APPROVED** [review]

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**